dags/kpi_forecasting.py

```diff
@@ -31,26 +31,7 @@
 tags = [Tag.ImpactTier.tier_3]
```

Comment:
Increasing priority, since you mention it is at least high priority for some part of the month.
```suggestion
tags = [Tag.ImpactTier.tier_1]
```